mu/virtual_environment.py

```diff
@@ -618,10 +703,10 @@ def create(self):
         before re-raising the error.
         """
         self.create_venv()
-        self.upgrade_pip()

```

Comment:
If `upgrade_pip()` is now dead code we might as well remove it as well.

Reply:
Done